### DISCUSSION TITLE
Donate

### DISCUSSION THREAD

**Discussion by yamiteru** [JonasKruckenberg/imagetools#8] [General]
As far as I can tell the v3 will become the best image compression plugin for vite. So I'd like to support this plugin via donation. I think nowadays the best place for that is https://opencollective.com.

**Comment by JonasKruckenberg** [comment]
Hey mate, thanks so much! 
To be honest I was a bit surprised by because I'd never thought someone would actually sponsor me, but that awesome!
I set up sponsorship through GitHub, with open collective in the making.
But again thanks so much for even considering this! 🙏

**Comment by yamiteru** [comment]
I'll wait till the open collective is set up since I use it for donating to all the projects I like. I hope other people will find this plugin useful and will donate as well. After I finish my project which heavily uses this plugin I'll definitely shout this plugin out.

Until then I'll probably be submitting issues and maybe later on when the first version of v3 is out I might contribute to the code as well if needed.

**Comment by JonasKruckenberg** [comment]
Yes that sounds awesome! I'm definitely looking forwards to seeing what you end up building too!
Maybe we could even have a featured projects section or something 🤔

In the meantime, I'm currently working on improved documentation, so if you stumble upon anything that's not quite clear or straight up wrong don't hesitate to open an issue! 
It's always easy to miss things when you already know your code 😅

**Reply by yamiteru** [reply]
Featured projects section is a good idea. Although GitHub shows where the library is used automatically on the right hand side like this:

<img width="360" alt="image" src="https://user-images.githubusercontent.com/34185465/110621582-ad74ed00-819a-11eb-905d-e5407f875ea3.png">

Once v1 version of my website is up and running I'll definitely let you know 👍

Currently I check this repo at least once a day and I think about ways of improving it even further so I'll definitely be active here in discussions. Btw good idea to move discussions to discussions 👌

**Reply by JonasKruckenberg** [reply]
Thanks! I believe that optimizing images is at least as important as optimizing your javascript, so I'd like to promote projects that do very well in that regard. Therefore I think a featured section would still be a great thing to have.
Regarding the open collective you proposed, they only accept GitHub projects with 100 or more stars, something that this plugin is miles away from 😅 I'll have a bit of a look around their website though.

**Reply by yamiteru** [reply]
Yeah I agree 👍

Oh I see.. I'll have to set up paypal then.

**Comment by yamiteru** [comment]
Can't you set up something like this? https://github.com/sponsors/preactjs

**Reply by JonasKruckenberg** [reply]
I actually do for myself https://github.com/sponsors/JonasKruckenberg 😄 Having one for imagetools would require creating a whole organization for it and all.
This might make sense if imagetools is used by more people at some point, but for now I think it's a bit overkill isn't it?

**Reply by yamiteru** [reply]
Nice! So I'm gonna sponsor you directly. Yes setting up an organisation is overkill.
